devenv/.gitignore

```diff
@@ -0,0 +1,3 @@
+cloud
+K8S
+local-commands.sh
```

Comment:
Does this file contain things that would be useful for us? It could be checked in to a `scripts` folder then.

Reply:
It does but it will be push in the next PR